api/restHandler/AppListingRestHandler.go

```diff
@@ -571,7 +661,7 @@ func (handler AppListingRestHandlerImpl) FetchAppStageStatus(w http.ResponseWrit
 	}
 	//RBAC enforcer Ends
 
-	triggerView, err := handler.appListingService.FetchAppStageStatus(appId)
+	triggerView, err := handler.appListingService.FetchAppStageStatus(appId, isJob)
```

Comment:
since this API is used only in case deployable Apps so pass default value as false